### ISSUE TITLE
ANSI escape sequences don't work with PDF outputs

### ISSUE THREAD

**Issue by mgeier** [jupyter/nbconvert#1141]
Example notebooks can be found at https://github.com/spatialaudio/nbsphinx/tree/master/doc

For example, this:

    python3 -m nbconvert --execute --to pdf allow-errors-per-cell.ipynb

... produces this:

![image](https://user-images.githubusercontent.com/705404/68601949-b15d0600-04a5-11ea-81d9-1defa23b9b9b.png)

... while it should look a bit more like this (which was produced by `nbsphinx`'s LaTeX output):

![image](https://user-images.githubusercontent.com/705404/68602361-7d361500-04a6-11ea-9fc2-f371695ee8e7.png)

Those two don't even create PDF files, they lead to some errors during the LaTeX run:

    python3 -m nbconvert --to pdf pre-executed.ipynb
    python3 -m nbconvert --execute --to pdf code-cells.ipynb

**Nbconvert version:** `master`

Possibly related:
#1068, #1071, #1077

**Comment by MSeal** [comment]
I'll need to double check what went wrong there, thanks for raising. If you find the offending line for the raises-exception feel free to add the fix in a PR. The LaTeX story is more of a mess and I haven't had time to try and help there for a while.

**Comment by aarchiba** [comment]
Could this be fixed by https://github.com/jupyter/nbconvert/pull/1181 ?

**Comment by mgeier** [comment]
@aarchiba Nope. #1181 was only a partial fix. The `block traceback_line` still has a `strip_ansi | escape_latex` which should be replaced with `escape_latex | ansi2latex`.
